src/services/submissionService.ts

```diff
@@ -175,6 +175,53 @@ export const updateSubmission = async (id: string, updatedFields: Partial<dbSche
             updatedDate: new Date().toISOString()
         }
 
+        if (updatedFields.isApproved === true) {
+            const facilityData: gqlType.FacilityInput = {
+                nameEn: submissionToUpdate.healthcareProfessionalName,
+                nameJa: submissionToUpdate.healthcareProfessionalName,
+                contact: {
+                    email: '',
+                    phone: '',
+                    website: '',
+                    mapsLink: submissionToUpdate.googleMapsUrl,
+                    address: {
+                        postalCode: '',
+                        prefectureEn: '',
+                        cityEn: '',
+                        addressLine1En: '',
+                        addressLine2En: '',
+                        prefectureJa: '',
+                        cityJa: '',
+                        addressLine1Ja: '',
+                        addressLine2Ja: ''
+                    }
+                },
+                healthcareProfessionals: [],
+                isDeleted: false
+            }
```

Comment:
@ermish @theyokohamalife 
This is my main area of concern. Currently, when we pass the submission ID and mark the submission as `isApproved`, a new facility and healthcarePro are created, which is great. However, there are several fields which are hardcoded, and only the fields from `AddSubmissionInput` are added to the the new facility/healthcarePro (`googleMapsUrl`, `healthcareProfessionalName`, and `spokenLanguages`).

In my mind, the user completes the basic `addSubmission` form with the three `AddSubmissionInput` fields, the moderator/admin approves it, and the submission gets added to the DB as a new facility/healthcarePro. But what about the other fields (contact, specialty, etc.)? Do they come form the admin who approves? Or is the user also meant to include this extra info? Any guidance or advice would be mush appreciated 🙏

Reply:
@evan-desu The other fields such as contact, specialty, etc are entered by the moderator after confirming the information with the healthcare provider.

Reply:
@theyokohamalife Thank you for confirming that for me!

Reply:
After rebasing, `updateSubmission` no longer adds a facility or healthcarePro, most likely due to changes made to `addFacility` and/or `addHealthcareProfessional`.

Reply:
update: I've changed the logic to be where the moderator should fill out the facility/provider before approving. This will prevent incomplete results to show up in user searches